ci-lib-core/src/sql.rs

```diff
@@ -197,10 +197,10 @@ pub const CREATE_JOBS_TABLE: &'static str = "\
 
 pub const CREATE_METRICS_TABLE: &'static str = "\
     CREATE TABLE IF NOT EXISTS metrics (id INTEGER PRIMARY KEY AUTOINCREMENT,
-        job_id INTEGER,
+        run_id INTEGER,
         name TEXT,
         value TEXT,
-        UNIQUE(job_id, name)
+        UNIQUE(run_id, name)
```

Comment:
<img width="340" alt="image" src="https://github.com/user-attachments/assets/8b875552-97b9-4736-980c-d6f2c8296b9a" alt="a screenshot of sqlite3 dumping the schema for a metrics table on a host named ci-butactuallyin-space. it also has this column renamed to run_id." />

oops. perfect rename here. thank you.